### DISCUSSION TITLE
Monitor VMs and restart any that are unresponsive...

### DISCUSSION THREAD

**Discussion by Doni49** [tteck/Proxmox#2887] [Ideas]
When I first started using ProxMox about a year ago, I found your scripts and tried out the one that monitors all the VMs.  When it finds one that has become unresponsive, it rebooted ALL of them.  So I removed it.

I would LOVE to have a script that does similar but just restarts the specific VM that is unresponsive instead of all of them.

**Comment by tteck** [comment]
>  just restarts the specific VM that is unresponsive instead of all of them.

The script is designed to only restart the affected LXC/VM, not all.
